fastapi/dependencies/utils.py

```diff
@@ -685,6 +678,16 @@ def request_params_to_args(
     return values, errors
 
 
+def get_missing_field_error(field_alias: str) -> ErrorWrapper:
```

Comment:
Since this error is now used in two places, I added a utility function to generate it.